pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -40,6 +40,7 @@
     public static final String LEAD_CONTROLLER_RESOURCE_NAME = "leadControllerResource";
 
     public static final String LEAD_CONTROLLER_RESOURCE_ENABLED_KEY = "RESOURCE_ENABLED";
+    // TODO: Rename this config because it applies to both SQL and PQL
     public static final String ENABLE_CASE_INSENSITIVE_PQL_KEY = "enable.case.insensitive.pql";
```

Comment:
We can add something like `enable.case.sensitivity` and fall back to `enable.case.insensitive.pql`? Since it's a pretty simple change, we can include to this PR?

Reply:
Done